### DISCUSSION TITLE
How to reproduce the behavior of the stringify() method | @vueuse/docs-utils

### DISCUSSION THREAD

**Discussion by maxlestage** [vueuse/vueuse#3708] [Q&A] [answered]
I have a problem with one of the examples it is not possible to use an old vueuse method import { stringify } from '@vueuse/docs-utils' https://github.com/vueuse/vueuse/blob/main/packages/core/useStorage/demo.vue

Do you know what I can replace this stringify with in order to be able to see the changes live as in the example: https://vueuse.org/core/useStorage/

**Comment by hotenov** [comment, marked as answer]
@maxlestage Hi! '`@vueuse/docs-utils`' this is a path alias for TS configuration, which you can find in '[tsconfig.json](https://github.com/vueuse/vueuse/blob/main/tsconfig.json)'

So, the answer on your question will be the source code of target file:

```typescript
import { reactify } from '@vueuse/shared'
import YAML from 'js-yaml'

export const stringify = reactify(
  (input: any) => YAML.dump(input, {
    skipInvalid: true,
    forceQuotes: true,
    condenseFlow: true,
    noCompatMode: true,
    quotingType: '\'',
  }),
)
```

To use it in your code you also need to install "js-yaml" package

```plain
npm install js-yaml
```
